app/main/views/frameworks.py

```diff
@@ -330,3 +331,97 @@ def get_framework_interest(framework_slug):
     supplier_ids = [supplier_framework.supplier_id for supplier_framework in supplier_frameworks]
 
     return jsonify(interestedSuppliers=supplier_ids), 200
+
+
+@main.route('/frameworks/transition-dos/<string:framework_slug>', methods=['POST'])
+def transition_dos_framework(framework_slug):
+    updater_json = validate_and_return_updater_request()
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ['expiringFramework'])
+
+    going_live_framework = Framework.query.filter(
+        Framework.slug == framework_slug
+    ).first_or_404()
+    expiring_framework = Framework.query.filter(
+        Framework.slug == json_payload['expiringFramework']
+    ).first_or_404()
+
+    if going_live_framework.id <= expiring_framework.id:
+        abort(400, f"'going_live_framework' ID ('{going_live_framework.id}') must greater than"
+              f"'expiring_framework' ID ('{expiring_framework.id}')")
+
+    if any(f.framework != 'digital-outcomes-and-specialists' for f in (going_live_framework, expiring_framework)):
+        abort(400, f"'going_live_framework' family: '{going_live_framework.framework}' and 'expiring_framework' family:"
+              f" '{expiring_framework.framework}' must both be 'digital-outcomes-and-specialists'")
+
+    if going_live_framework.status != 'standstill' or expiring_framework.status != 'live':
+        abort(400, f"'going_live_framework' status ({going_live_framework.status}) must be 'standstill', and "
+              f"'expiring_framework' status ({expiring_framework.status}) must be 'live'")
+
+    expiring_framework_draft_briefs = Brief.query.filter(
+        Brief.status == 'draft',
+        Brief.framework_id == expiring_framework.id,
+    ).all()
```

Comment:
Again, should probably lock all of these rows `.with_for_update()` else there's a possibility we could clobber an update made to the brief after we've fetched it but before we issued the `UPDATE` to it in the loop below.